src/ripple/app/main/Application.cpp

```diff
@@ -850,74 +847,26 @@ class ApplicationImp : public Application, public RootStoppable, public BasicApp
     bool
     initSQLiteDBs()
```

Comment:
Should change the name of this function, since the DBs might not be SQLite

Reply:
Done.